anoncreds/src/data_types/anoncreds/macros.rs

```diff
@@ -0,0 +1,5 @@
+macro_rules! ursa_cl {
```

Comment:
Very unfamiliar with this specific macro, but why can't we directly just import the package?

Reply:
Ah I see that it came from another macro which was based on a config on whether cl/cl_native was enabled.

Right now, this macro can be removed and it can be imported directly as we always pull in cl/cl_native right now.

Reply:
Good catch.

Reply:
Removed